src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASI/Arguments/DirectoryArgument.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasi;
+
+internal class DirectoryArgument : RequiredStringArgument
```

Comment:
Instead of this we should use 

https://github.com/dotnet/xharness/blob/1e1eab570c2177aa9dfb2de2826dfa28aa456f8c/src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/WasmTestBrowserCommandArguments.cs#L13-14

https://github.com/dotnet/xharness/blob/1e1eab570c2177aa9dfb2de2826dfa28aa456f8c/src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Arguments/AppPathArgument.cs#L10-L15

- That argument is for specifying the directory that contains the application, and that's what we are trying to specify with `wasmtime --dir` too.
- Also, from the user pov it will be consistent across wasm/wasi/others.